src/main/resources/templates/admin/index.html

```diff
@@ -149,6 +149,22 @@ <h2>Add Organization User</h2>
         </form>
     </th:block>
 
+    <th:block th:if="${canRemoveOrganizationUser}">
+        <h2>Remove Organization User</h2>
+
```

Comment:
Maybe add a blurb here to note that you can only remove users who are listed as TF contacts for the organization. If you try to remove a regular user, I believe the permission check will fail.